tests/tags.spec.js

```diff
@@ -238,6 +238,22 @@ describe('Tags', function() {
         bar: 'baz'
       });
     });
+
+    it('should not overwrite footers with the same key', function() {
+      const footer = tags.parseFooterTagLines([
+        'Foo: bar',
+        'Foo: bar',
+        'Bar: baz',
+        'Foo: baz'
+      ]);
+
+      m.chai.expect(footer).to.deep.equal({
```

Comment:
Would `{Foo: ['bar', 'bar', 'baz'], Bar: 'baz'}` not be a better output?

Reply:
I guess that wouldn't allow you to tell that `'Bar: baz'` occurs before the last `'Foo: baz'`, but I dunno if that's important or not?

I guess a downside to @nazrhom 's proposed output is that you won't be able to tell the difference between inputs of
```
[
    'Foo: bar',
    'Foo: bar',
    'Bar: baz',
    'Foo: baz'
]
```
and
```
[
    'Foo: bar',
    'Foo1: bar',
    'Bar: baz',
    'Foo2: baz'
]
```
but again I've got no idea if that's important or not.